osd/invalid_iaas_credentials.json

```diff
@@ -3,6 +3,6 @@
     "service_name": "SREManualAction",
     "cluster_uuid": "${CLUSTER_UUID}",
     "summary": "Action required: delete inaccessible cluster",
-    "description": "Your cluster requires you to take action because Red Hat OpenShift Cluster Manager is not able to access the infrastructure with the provided credentials. If you intended to terminate this cluster then please delete the cluster, otherwise file a support request.",
+    "description": "Your cluster requires you to take action because Red Hat SRE is not able to access the infrastructure with the provided credentials. Please restore the credentials and permissions provided during install, or file a support request.",
```

Comment:
Should remove "SRE" if we're changing from OpenShift Cluster Manager. It doesn't matter what can't use the infrastructure and avoid references to humans unless it's relevant.